### PR TITLE
White toolbar component

### PR DESCRIPTION
**Pull request by raonismaneoto**
**Feature/Bug description:**
Wrap all white toolbars into a component
**Solution:**
The following screens has already been covered:

- My profile

- Edit institution's info

- Search institution

screenshot from 2019-02-20 14-39-06

screenshot from 2019-02-20 14-39-22

screenshot from 2019-02-20 14-39-38

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/toolbar/white_toolbar_mobile.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div class="white-toolbar-mobile" ng-if="whiteToolbarCtrl.isMobileScreen()">
+    <md-button class="md-icon-button" ng-click="whiteToolbarCtrl.goBack()">
+        <md-icon class="toolbar-icon-alignment">{{whiteToolbarCtrl.primaryButtonIcon}}</md-icon>
+    </md-button>
+    <div style="grid-column-start: 2; grid-column-end: 3;"
```
Comment (ruanres) [inline frontend/toolbar/white_toolbar_mobile.html]: move style to css

File: frontend/user/configProfile/configProfileController.js [hunk 2]
```diff
@@ -181,6 +182,17 @@
             window.history.back();
```
Comment (ruanres) [inline frontend/user/configProfile/configProfileController.js:182]: remove this function, once it is not used anymore

File: frontend/toolbar/white_toolbar_mobile.html (hunk 1 above)
Comment (JuliePessoa) [inline frontend/toolbar/white_toolbar_mobile.html]: css to css file

File: frontend/toolbar/white_toolbar_mobile.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            <md-button class="md-icon-button" ng-click="$mdMenu.open(ev)">
+                <md-icon class="toolbar-icon-alignment">more_vert</md-icon>
+            </md-button>
+            <md-menu-content style="width: 100%;">
+                <md-menu-item ng-repeat="option in whiteToolbarCtrl.menuOptions" ng-click="option.action()">
+                    <div>
+                        <md-icon>
+                            {{option.icon}}
+                        </md-icon>
+                        <span style="margin-left: 0.5em;">{{option.title}}</span>
```
Comment (JuliePessoa) [inline frontend/toolbar/white_toolbar_mobile.html]: css to css file

File: frontend/user/configProfile/configProfileController.js [hunk 4]
```diff
@@ -181,6 +182,17 @@
...
         };
 
+        /**
+         * Sets save button's properties.
+         */
+        configProfileCtrl.setSaveButton = () => {
+            configProfileCtrl.saveButton = {
+                class: 'config-profile__toolbar--save',
+                action: configProfileCtrl.finish,
+                name: 'SALVAR'
```
Comment (ruanres) [inline frontend/user/configProfile/configProfileController.js]: add the option to disable the button

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

File: frontend/toolbar/white_toolbar_mobile.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        <md-icon ng-if="whiteToolbarCtrl.rightButton['icon']">
+            {{whiteToolbarCtrl.rightButton.icon}}
+        </md-icon>
+    </md-button>
+    <div ng-if="whiteToolbarCtrl.menuOptions" class="white-toolbar-menu-button">
+        <md-menu>
+            <md-button class="md-icon-button" ng-click="$mdMenu.open(ev)">
+                <md-icon class="toolbar-icon-alignment">more_vert</md-icon>
+            </md-button>
+            <md-menu-content style="width: 100%;">
```
Comment (ruanres) [inline frontend/toolbar/white_toolbar_mobile.html]: style to css

File: frontend/toolbar/white_toolbar_mobile.html (hunk 3 above)
Comment (ruanres) [inline frontend/toolbar/white_toolbar_mobile.html]: same here

File: frontend/toolbar/white_toolbar_mobile.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  grid-template-columns: 20% 50% auto;
+  align-items: center;
+}
+
+.toolbar-icon-alignment {
+    font-size: 2.5em;
+    line-height: inherit !important;
+}
+
+.edit-inst-toolbar-title {
```
Comment (ruanres) [inline frontend/toolbar/white_toolbar_mobile.css:18]: use the property text-transform to set the title to make sure it will always be uppercase

**Review by ruanres — CHANGES_REQUESTED** [review]

File: frontend/user/configProfile/configProfileController.js (hunk 2 above)
Comment (raonismaneoto) [reply frontend/user/configProfile/configProfileController.js:182]: actually it is being used at config_profile_mobile.html. I guess it is useful when a user tries to get in the other user's profile page

**Review by ruanres — APPROVED** [review]

**Review by JuliePessoa — APPROVED** [review]

**Pull request merged**